### PR TITLE
LPS-173776, LPS-173777, LPS-173778, LPS-173779 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Tickets:**
* [LPS-173776](https://issues.liferay.com/browse/LPS-173776)
* [LPS-173777](https://issues.liferay.com/browse/LPS-173777)
* [LPS-173778](https://issues.liferay.com/browse/LPS-173778)
* [LPS-173779](https://issues.liferay.com/browse/LPS-173779)

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase [hunk 1]
```diff
@@ -499,6 +499,112 @@ definition {
 		}
 	}
 
+	@description = "Verify if is not possible navigate through pages on public pages with not contains validation error"
+	@priority = 5
+	test CannotNavigateThroughThePagesOnPublicPagesWithDoesNotContainsValidationError {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
```
Comment (andreefarias) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase:506]: Remove this property: `property portal.upstream = "true";`

Acceptance property replaces the behavior for it.

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**